### PR TITLE
Feature/homepage destinations

### PR DESCRIPTION
**Pull request by flibbertigibbet**
* Redirect to directions to clicked homepage featured destination
* Implement 'view all' featured locations button on homepage
* Fix destinations serialization on search endpoint

### CODE REVIEW

File: src/app/scripts/cac/pages/cac-pages-home.js [hunk 1]
```diff
@@ -129,4 +200,37 @@ CAC.Pages.Home = (function ($, UserPreferences) {
...
+     * Convert destination from search endpoint into a feature formatted like typeahead results.
+     * TODO: Featured destinations should probably be in the typeahead, and this function
+     * moved or removed.
+     *
+     * @param {Object} destination JSON object returned from destination search endpoint
+     * @returns {Object} Feature object structured like the typahead results, for use on map page.
+     */
+    function convertDestinationToFeature(destination) {
+        var feature = {
+            name: 'Philadelphia City Hall',
```
Comment (CloudNiner) [inline src/app/scripts/cac/pages/cac-pages-home.js]: Doesn't the destination have a name or title field?

Comment (flibbertigibbet) [inline src/app/scripts/cac/pages/cac-pages-home.js]: Oh, whoops, missed that one.

File: src/app/scripts/cac/home/cac-home-templates.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    function destinations(destinations) {
+        var source = [
+            '{{#each destinations}}',
+            '<div class="col-sm-4">',
+            '<a class="block block-destination block-half" href="#">',
+            '<h3 class="destination-name">{{this.name}}</h3>',
+            '<h5 class="destination-address">{{this.address}}</h5>',
+            '<h5 class="destination-address-2">{{this.city}}, {{this.state}} {{this.zip}}</h5>',
+            '<img src="{{#if this.wide_image}}{{ this.wide_image }}{{^}}http://placeholthis.it/300x150{{/if}}"',
```
Comment (CloudNiner) [inline src/app/scripts/cac/home/cac-home-templates.js]: placehold.it*

**Comment by CloudNiner** [comment]
LGTM

File: src/app/scripts/cac/home/cac-home-templates.js (hunk 2 above)
Comment (flibbertigibbet) [inline src/app/scripts/cac/home/cac-home-templates.js]: Hm.  I'd copied this from the itinerary list template, so I guess that one needs to be fixed, too.

Comment (flibbertigibbet) [inline src/app/scripts/cac/home/cac-home-templates.js]: search-and-replace derp

**Pull request merged**